### PR TITLE
Scatterplot skeleton

### PR DESCRIPTION
**Pull request by henrhaat**

### CODE REVIEW

File: components/scatterplot.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import dash_html_components as html
+import dash_core_components as dcc
+
+from utils import earthquake_data
+
+
+def get_graph(session_id):
+    """Returns an empty scatterplot graph, if data has been uploaded.
+    Otherwise returns a text informing the user, that no data was found.
```
Comment (llmlks) [inline components/scatterplot.py]: According to PEP8 function docstrings should be written in imperative ("Return an ...")

**Review by llmlks — CHANGES_REQUESTED** [review]
Looks good! Only a minor change in docs needed

**Comment by henrhaat** [comment]
Made the requested changes

File: components/scatterplot.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 
 def get_graph(session_id):
-    """Returns an empty scatterplot graph, if data has been uploaded.
+    """Return an empty scatterplot graph, if data has been uploaded.
     Otherwise returns a text informing the user, that no data was found.
```
Comment (llmlks) [inline components/scatterplot.py]: Sorry to nitpick, but could you change the other sentence, too?

**Review by llmlks — CHANGES_REQUESTED** [review]

**Comment by henrhaat** [comment]
Made the requested change

**Review by llmlks — APPROVED** [review]

**Pull request merged**